src/celestia/win32/winmain.cpp

```diff
@@ -1673,7 +1673,7 @@ void ShowWWWInfo(const Selection& sel)
             if (url.empty())
             {
                 char name[32];
-                sprintf(name, "HIP%d", sel.star()->getCatalogNumber() & ~0xf0000000);
+                sprintf(name, "HIP%d", sel.star()->getIndex() & ~0xf0000000);
```

Comment:
As you changed this line, could you replace `0xf0000000` with `Star::MaxTychoCatalogNumber`?